neuhatch/views.py

```diff
@@ -131,8 +131,14 @@ def search():
         max_results -- Integer, the most tweets to return
     """
     query = request.args.get('q')
+
+    if request.args.get('max_results') is None:
```

Comment:
:+1: I just realized this should be in `/search.csv` too, where should I put this to avoid writing it twice?